ci/gpu/build.sh

```diff
@@ -39,11 +39,15 @@ gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
 
-gpuci_logger "Install tests dependencies"
+gpuci_logger "Install conda dependencies"
 gpuci_mamba_retry install -y \
     "cuxfilter=${MINOR_VERSION}" \
     "faker" \
     "python-whois" \
+    "requests" \
+    "matplotlib" \
+    "faker" \
+    "transformers=4.*" \
```

Comment:
don't think we need to add these. `requests`,`matplotlib`, and `transformers` already on the images used for ci builds. `faker` already added.

Reply:
ok, I will remove them. thanks